src/switch_loadable_module.c

```diff
@@ -2854,6 +2854,19 @@ SWITCH_DECLARE(int) switch_loadable_module_get_codecs_sorted(const switch_codec_
 
 				if (imp->codec_type != SWITCH_CODEC_TYPE_VIDEO) {
 					uint32_t crate = !strcasecmp(imp->iananame, "g722") ? imp->samples_per_second : imp->actual_samples_per_second;
+					if (!strcasecmp(name, "amr-wb")) {
+						if (!zstr(fmtp) && imp->matches_fmtp) {
+							if (SWITCH_STATUS_SUCCESS == imp->matches_fmtp(fmtp, imp->fmtp)) {
```

Comment:
`matches_fmtp` is new callback right? I think we can remove the validation for amr-wb and only check for `matches_fmtp` is implemented. So in the future for example where we can also implemented matches_fmtp for OPUS, VP8 etc

Reply:
correct, will fix it